src/Module/Conversation/Network.php

```diff
@@ -120,7 +120,7 @@ protected function content(array $request = []): string
 			$content = '';
 
 			if (self::$forumContactId) {
-				// If self::$forumContactId belongs to a community forum or a privat goup,.add a mention to the status editor
+				// If self::$forumContactId belongs to a community forum or a private goup,.add a mention to the status editor
```

Comment:
The `,.` doesn't look great. 

I'm happy to fix things up if someone provides text (again, my default is to try to only change spelling, as even just the spelling changes are quite a lot to review)